api/Worker.json

```diff
@@ -160,8 +160,15 @@
               },
               "edge": "mirror",
               "firefox": {
-                "version_added": false,
-                "notes": "See <a href='https://bugzil.la/1247687'>bug 1247687</a>"
+                "version_added": "111",
+                "notes": "Available only in Nightly builds. See <a href='https://bugzil.la/1247687'>bug 1247687</a>",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "dom.moduleScripts.enabled",
```

Comment:
The correct pref is `dom.workers.modules.enabled`. https://searchfox.org/mozilla-central/rev/7bb059f2e81636e0db20acbf2afe67d49508489b/modules/libpref/init/StaticPrefList.yaml#3936

We also have #19044